### ISSUE TITLE
Initial TalkBack focus inconsistent

### ISSUE THREAD

**Issue by zwikk** [dotnet/maui#32632]
### Description

We are currently not meeting WCAG 2.2 guideline 1.3.2, which requires content to follow a meaningful sequence and ensures that the initial focus in the reading order is correct. For Android applications, this means the first TalkBack focus should be on the back button in the navigation bar, if it is visible.

Below illustrates the inconsistent behavior on Android.

### Steps to Reproduce

The **TalkBackTestApp** in the public repo illustrates the inconsistent behavior, which appears to be triggered by the introduction of a ScrollView control.

Steps to reproduce, first the correct and expected outcome.
1) Launch the app on an Android device
2) Start TalkBack
3) Double-tap button **Detail1 without ScrollView** to navigate
3) Observe the semantic focus is on the back button, this is correct.

The bug:
1) Navigate back to main page
2) Double-tap button **Detail2 with ScrollView** to navigate
4) Observe the semantic focus is on the page Title label. This is incorrect, the focus should be on the BackButton

It looks like the ScrollView in Detail2Page has a negative impact on the initial focus.

And another confusion result:
1) Navigate back to main page
2) Double-tap button **Detail3 without ScrollView** to navigate
3) Observe the semantic focus is on the back button, again this is correct.
4) Double-tap button **Detail3.1 with ScrollView** to navigate
5) Observe the semantic focus on the the back button, this is also correct.

Detail3.1 now includes a ScrollView, yet the initial focus remains correct. This outcome is unexpected and puzzling. Simply adding or removing a ScrollView doesn’t resolve the initial focus issue—there’s clearly more at play.

### Link to public reproduction project repository

https://github.com/zwikk/maui-playground/tree/main/Source/TalkBackTest

### Version with bug

9.0.110 SR12

### Is this a regression from previous behavior?

No, this is something new

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 16

### Did you find any workaround?

I don't have a workaround, adding or removing ScrollView does not simply solve the issue as demonstrated in the TalkBackTest app

### Relevant log output

```shell

```

**Comment by HarishKumarSF4517** [comment]
This issue has been validated in Visual Studio Code 1.106.0 with MAUI versions (9.0.0, 9.0.110, 9.0.120, 10.0.10, 10.0-rc2 and 10.0.0-preview7). It was not reproducible on Android platform. I have attached the video checked in multiple device for reference:

https://github.com/user-attachments/assets/6b104ca2-5785-4ec0-9392-01608bf0a8c2

https://github.com/user-attachments/assets/b3901e12-fea6-40f1-b03e-aae91bc80de6

https://github.com/user-attachments/assets/e58ac999-3c5f-4c71-b94d-1cc0b428b669

**Comment by zwikk** [comment]
That's unfortunate. Here's the repro on a Samsung Galaxy A53 with Android 16 build with .NET 9.0.306 and MAUI controls 9.0.120.

As you can see the focus of Detail2 with scrollview page is initially incorrect.

https://github.com/user-attachments/assets/1308eee4-e308-46d4-8f81-50f0f1cae865

**Comment by zwikk** [comment]
I'm seeing some different results on a Pixel 9A, the test app seems to work fine as well. I have added a number of pages and after some deep navigation, it starts to fail even on Pixel 9A. See attached video.

Steps to reproduce

1) Get latest version and run app
1) Start TalkBack
1) Double-tap **Detail2 with ScrollView**
1) Navigation occurs, initial focus correct
1) Double-tap **Detail2.1 with ScrollView**
1) Navigation occurs, initial focus incorrect
 
Repro source code has been updated.

https://github.com/user-attachments/assets/a7d97ef3-0a50-437d-a552-728a9bec7420

**Comment by zwikk** [comment]
I'm closing this issue because we technically met the WCAG requirement, as the correct behavior was confirmed on certain devices. Apparently, that level of consistency is considered sufficient for WCAG approval.
